README.md

```diff
@@ -33,9 +32,21 @@ python setup.py install
 
 ## Dependencies
 
-+ Python 2.7 / Python 3.3
-+ [Requests](https://github.com/requests/requests)
-+ [PyJWT](https://github.com/jpadilla/pyjwt)
+- Python 2.7 / Python 3.3
+- [Requests](https://github.com/requests/requests)
+- [PyJWT](https://github.com/jpadilla/pyjwt)
+
+## Versioning
+
+This client is versioned and released alongside App Search.
+
+To guarantee compatibility, use the most recent version of this library within the major version of the
+corresponding App Search implementation.
+
+For example, for App Search `7.3`, use `7.3` of this library or above, but not
+`8.0`.
+
+If you are a [cloud](app.swiftype.com/as) user, simply use the most recent version of this library.
```

Comment:
To avoid future name collision with App Search in Elastic Cloud, I would use the term SaaS here.